pom.xml

```diff
@@ -189,6 +189,36 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-dynamic-simulation-tool</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
```

Comment:
Why do we need to add actions related artifacts? It has nothing to do with dynamic simulation?

Reply:
Removing them. They were added by mistake, in order to add `GroovyDslContingenciesProviderFactory` which was spotted in `powsybl-action-dsl`. But in fact that factory is already in `powsybl-contingency-dsl`.